packages/expo-bluetooth-utils/build/Bluetooth.js

```diff
@@ -0,0 +1,32 @@
+import { Base64 } from 'js-base64';
```

Comment:
Would the need for this go away with Turbomodules? (Just thinking about a future version of the library..)